### PR TITLE
JAVA-20166 Migrating spring-date-eclipselink

### PR DESCRIPTION
**Pull request by timis1**
Migrating spring data related modules

### CODE REVIEW

File: persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/PersonRepositoryService.java [hunk 1]
```diff
@@ -23,11 +23,11 @@ public void setPersonRepository(PersonRepository repo) {
     }
 
     public Person findOne(String id) {
-        return repo.findOne(id);
+        return repo.findById(id).orElseGet(null);
```
Comment (dkapil) [inline persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/PersonRepositoryService.java:26]: can we simply return Optional?

File: persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/StudentRepositoryService.java [hunk 2]
```diff
@@ -23,11 +23,11 @@ public void setStudentRepository(StudentRepository repo) {
     }
 
     public Student findOne(String id) {
-        return repo.findOne(id);
+        return repo.findById(id).orElseGet(null);
```
Comment (dkapil) [inline persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/StudentRepositoryService.java:26]: can we return optional?

**Review by dkapil — CHANGES_REQUESTED** [review]

File: persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/PersonRepositoryService.java (hunk 1 above)
Comment (timis1) [reply persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/PersonRepositoryService.java:26]: Done, I returned an Optional.

File: persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/StudentRepositoryService.java (hunk 2 above)
Comment (timis1) [reply persistence-modules/spring-data-couchbase-2/src/main/java/com/baeldung/spring/data/couchbase/service/StudentRepositoryService.java:26]: Done, I returned an Optional.

**Pull request merged**